tox.ini

```
@@ -82,6 +82,23 @@ commands =
         {toxworkdir}/cpython/Lib
     pytest -v docs/tests/test_standard_library_docs.py
 
+[testenv:numpy-apidocs]
+description = Build numpy API documentation
```

Maybe add a short info to introduce the purpose and scope of this test.

```suggestion
description = Build numpy API documentation. For now we don't check for any warnings or other errors. The only purpose of this test is to make sure pydoctor doesn't crash.
```